Error: The `form` element was not requested.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group/1/b8ab23c5-9980-488d-b3d1-6676bb27a526.html b/output/Checkbox Group/1/b8ab23c5-9980-488d-b3d1-6676bb27a526.html
index 7698747..c173a45 100644
--- a/output/Checkbox Group/1/b8ab23c5-9980-488d-b3d1-6676bb27a526.html	
+++ b/output/Checkbox Group/1/b8ab23c5-9980-488d-b3d1-6676bb27a526.html	
@@ -1,8 +1,15 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label><br>
-    <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label><br>
-    <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>What fruits do you like?</legend>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
+</fieldset>
